File: PPA/Assignments/switch Assignment/Programs/question3.c
#include<stdio.h>
int main(){
	float a = 5.2;
	switch((int)a){
		case 1:
			printf("one\n");
			break;
		case 5:
			printf("five\n");
			break;
		case 2:
			printf("two\n");
			break;
		default:
			printf("else\n");
			break;
		
}

/*
output:
	question3.c: In function ‘main’:
	question3.c:17:1: error: expected declaration or statement at end of input
   	17 | }
   	   | ^
-------------------------------------------------------------------------------------------
Exp:
	'}' is required at the end of the switch block 	
*/

File: Java/Assignments/operator 2 Assignment/programs/question3.java
/*
what will be the output of the statement if i=5, j=4
	1.(i++ >= j++) && (--j < i++)
	2.(--i < j--) || (j++ >= ++i)
*/

class Question3{
	public static void main(String[] arg){
		int i=5,j=4;
		boolean ans;
		
		ans=(i++ >= j++) && (--j < i++);
		/*
			ans=(i++ >= j++) && (--j < i++);
			ans=(5 >= 4) && (5 < 6)
			ans=true && true
			ans=true
		*/
		System.out.println(ans);
		
		
		ans=(--i < j--) || (j++ >= ++i);
		/*
			i=7, j=4
			
			ans=(--i < j--) || (j++ >= ++i);
			ans=(6 < 4) ||(3 >= 7)
			ans=false ||false
			ans=false
		*/
		System.out.println(ans);
	}
}
